src/py42/_internal/session.py

```diff
@@ -176,8 +176,6 @@ def _renew_authentication(self, use_cache=False):
 
     def _print_request(self, method, url, params=None, data=None):
         debug.logger.info(u"{0}{1}".format(str(method).ljust(8), url))
-        if self.headers:
```

Comment:
I removed this for two reasons:
- Security reasons (basic credentials and tokens were being logged)
- When updating to py42 1.5.0, if you were using a level of `DEBUG` (as the cli does), your logging became more verbose than it was previously (removing this makes it the same).

Reply:
Could we add a new `debug.log_headers` setting to make it possible but also need it set explicitly?